src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -666,7 +671,7 @@ public OptionData addChoice(@Nonnull String name, @Nonnull String value)
      * @throws IllegalArgumentException
      *         If any of the following checks fail
      *         <ul>
-     *             <li>The {@link OptionType} does {@link OptionType#canSupportChoices() support choices}</li>
+     *             <li>The {@link OptionType} does {@link OptionType#canSupportChoices() support choices} and some are provided.</li>
```

Comment:
```suggestion
     *             <li>The {@link OptionType} does {@link OptionType#canSupportChoices() support choices} and some are provided.</li>
```
Imo this sounds really vague, not sure how to phrase it differently though